### PR TITLE
402 notifications page

### LINKED ISSUE #402: Create page to see notification history
- This is the same feature that used to be available in the old version, basically a page that lists all notifications
- While we're at it, consider the following changes to the notification modal:
  - "Delete all notifications" should say "Clear notifications" or "Mark all as read" instead. I think this better conveys the intention of this button.
  - Remove the "Close" button since there is already an "X" button above. Instead, repurpose it to serve as a link to the new notification history page.

### PR DESCRIPTION
**Pull request by keyxcode**
closes #402

### CODE REVIEW

File: langcorrect/templates/notifications/partials/notification_loop.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+{% load humanize %}
+
+{% for notification in notifications %}
```
Comment (danielzeljko) [inline langcorrect/templates/notifications/partials/notification_loop.html:4]: We should make the parent do the actual looping and keep the child to just render itself. This keeps the child component atomic allowing us to render this component anywhere within the app without any unwanted side effects. 

Parent

```
<div class="list-group list-group-flush mb-3">
   forloop
      include this template
      ....
   endfor
</div>
```

Let's rename the file from notification_loop.html to notification.html

**Review by danielzeljko — CHANGES_REQUESTED** [review]
One small change needed, but nicely done!

File: langcorrect/templates/notifications/partials/notification_loop.html (hunk 1 above)
Comment (keyxcode) [reply langcorrect/templates/notifications/partials/notification_loop.html:4]: Great suggestion! Let me work on it.

**Review by danielzeljko — APPROVED** [review]
Looks good!

**Pull request merged**